src/System.Windows.Forms/src/System/Windows/Forms/ToolStripControlHost.cs

```diff
@@ -15,29 +13,29 @@ namespace System.Windows.Forms
     /// <summary>
     ///  ToolStripItem that can host Controls.
     /// </summary>
-    public class ToolStripControlHost : ToolStripItem
+    public partial class ToolStripControlHost : ToolStripItem
     {
-        private Control control;
-        private int suspendSyncSizeCount = 0;
-        private ContentAlignment controlAlign = ContentAlignment.MiddleCenter;
-        private bool inSetVisibleCore = false;
-
-        internal static readonly object EventGotFocus = new object();
-        internal static readonly object EventLostFocus = new object();
-        internal static readonly object EventKeyDown = new object();
-        internal static readonly object EventKeyPress = new object();
-        internal static readonly object EventKeyUp = new object();
-        internal static readonly object EventEnter = new object();
-        internal static readonly object EventLeave = new object();
-        internal static readonly object EventValidated = new object();
-        internal static readonly object EventValidating = new object();
+        private Control _control;
```

Comment:
```suggestion
        private readonly Control _control;
```
We can remove `_control = null` in `Dispose` method.

Reply:
This would be behaviour changing